packages/core/src/components/forms/numericInput.tsx

```diff
@@ -166,6 +174,7 @@ export interface INumericInputProps extends IIntentProps, IProps {
 
 export interface INumericInputState {
     currentImeInputInvalid: boolean;
+    locale?: string;
```

Comment:
why are you storing this here? It looks like you don't do anything with it. also it's not good practice to have a state field named the same as a prop. I think this should be removed.

Reply:
removed